src/pages/TechSafetyAndPrivacy.jsx

```diff
@@ -47,12 +66,29 @@ function TechSafetyAndPrivacy() {
 					},
 				]}
 			/>
-			<div className="pl-80">
+
+			<div className="md:pl-80">
 				<TechSafetyAndPrivacyIntro
 					dataFromSafetyAndPrivacyIntro={dataFromSafetyAndPrivacyIntro}
 					dataFromFirebase={dataFromFirebase}
 				/>
-				<YouTubeVideoSection osvalue={osvalue} />
+
+				<Box style={{ margin: 'auto', width: '70%', paddingBottom: '2rem' }}>
+					<Searchbar tagsFromSearchBar={tagsFromSearchBar} tags={tags} />
+					<Breadcrumb subtopicValue={subtopicValue} handleResetSubtopic={handleResetSubtopic} subtopics={subtoptics} />
+				</Box>
+
+				{subtoptics.length == 0 || subtopicValue.length > 0 || tags.length > 0 ? (
+					<YouTubeVideoSection
+						osvalue={osvalue}
+						subtopicValue={subtopicValue}
+						tags={tags}
+					/>
+				) : (
+					<SubtopicSelection
+						dataFromSubtopicSelector={dataFromSubtopicSelector}
+						subtopics={subtoptics} />
+				)}
```

Comment:
this piece of code seems to be repeated numerous times, are we doing this for easier modifiability? Otherwise I suggest, let's extract it out and avoid code repeat.